module/sidebar/components/food-buff-component.js

```diff
@@ -0,0 +1,65 @@
+import Component from '../../api/front-end/lib/component.js';
+
+export default class FoodBuffComponent extends Component {
+    constructor(store) {
+        super({
+            store,
+            element: $('#food-buff')
+        })
+        this._hidden = false;
+    }
+
+    /**
+     * React to state changes and render the component's HTML
+     * 
+     * @returns {void}
+     */
+    async render() {
+        if (!this.state.actor) return;
+        let output = "<img class='divider-image' src='systems/ptu/images/icons/DividerIcon_FoodBuff.png' style='border:none; width:200px;'>";
+        const foodBuffs = this.state.actor.system.digestionBuff.split(", ");
+        if (foodBuffs.length > 0) {
+            for (const foodBuff of foodBuffs) {
+                if (foodBuff != "None" && foodBuff != "")
+                {
+                    const foodBuffItem = game.ptu.data.items.find(i => i.name.toLowerCase().includes(foodBuff.toLowerCase()));
+                    if (foodBuffItem) {
+                        output += await renderTemplate("/systems/ptu/module/sidebar/components/food-buff-component.hbs", {
+                            name: foodBuffItem.name,
+                            img: foodBuffItem.img,
+                            id: foodBuffItem.id,
+                            color: 'gray',
+                            effect: foodBuffItem.system.effect,
+                            owner: this.state.actor.id
+                        });
+                    }
+                }
+            }
+        }
+        this.element.html(output);
```

Comment:
Should the foodbuff divider be shown if there is no food buff?

Reply:
I like having it there because it reminds players that they can feed their pokemon to give them food buffs.

My players often forget that this mechanic exists.